Remove option for accordion behavior from expandables documentation

marteki opened the pull request:
As shared in https://github.com/cfpb/capital-framework/issues/536 and https://github.com/cfpb/capital-framework/issues/665, we’ve removed the accordion option from our design patterns. This updates our documentation to match the implementation guidelines.

## Removals

- Remove the description of the (buggy, not used) accordion behavior for expandables

## Review

- @ielerol 
- @anselmbradford 
- @Scotchester 

## Screenshots

### Documentation before changes

![screen shot 2018-02-21 at 2 31 34 pm](https://user-images.githubusercontent.com/1183435/36501040-071f7d06-1714-11e8-8b8a-3fa4f7c33259.png)

### Documentation after changes

![screen shot 2018-02-21 at 2 30 29 pm](https://user-images.githubusercontent.com/1183435/36501066-1f134032-1714-11e8-94e8-823355b66d25.png)

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

marteki committed:
Remove option for accordion behavior from expandables documentation

As shared in https://github.com/cfpb/capital-framework/issues/536 and
https://github.com/cfpb/capital-framework/issues/665, we’ve removed the
accordion option from our design patterns. This updates our
documentation to match the implementation guidelines.

marteki committed:
Update package.json and package-lock.json

marteki committed:
Change dependency to `grunt-contrib-uglify-es`

marteki committed:
Updates to CSS and JS from Capital Framework

marteki commented:
Whew! The build passed in Travis now. Needed to shift one of the dependencies and point it at a different location. Thanks to @Scotchester for his help on troubleshooting that.

Scotchester commented on package.json:
```diff
@@ -32,6 +32,7 @@
     "cf-pagination": "^4.2.2",
     "cf-tables": "^4.2.1",
     "cf-typography": "^4.6.0",
+    "grunt-contrib-uglify-es": "^3.3.0",
```
This should replace `grunt-contrib-uglify` in the `devDependencies` list, and `uglify-es` can also be removed now.

marteki committed:
Correction to package placement in `package.json`

Updated package ended up in `dependencies` instead of
`devDependencies`. Corrected that.

marteki committed:
Remove `uglify.es`

Scotchester reviewed (approved):
This is totally fine now, but just to ask the question: Should we remove the Interactions section entirely, now? It seems a bit superfluous without the Accordion content.

marteki commented:
@Scotchester Hmm. Maybe, if it's merged with the State section. I feel odd about having such an extensive Style section without explicitly calling out the Interactions, since it's an interactive web element.

Alright, I'll add that piece as a TODO to the list on our internal design patterns tracking page, and merging this.

sebworks commented:
Hmmm, we are using the accordion on the OAH pages ( closing-disclosure / loan-estimate ) and I'm not in favor of removing them.  The accordion allows you manage long content on mobile, without forcing the user to scroll or close other expandables.

marteki commented:
@sebworks There's been a good bit of discussion about this already over the last several months, where designers and developers came to an understanding about the removal of the accordions. For more details and to catch up, see the summary at https://[GHE]/CFPB/hubcap/wiki/Design-pattern-additions-and-changes#expandable-accordions-removed.

The current plan was to adjust the expandable groups for the one known place where accordions are being used, but the Loan Estimate page was not previously called out as a place! Let's discuss it more internally.

(Somewhat related, and an issue you were previously tagged on for comment that might have gotten buried in your inbox: https://github.com/cfpb/capital-framework/issues/536)

ielerol reviewed (approved):
Looks good to me!

marteki commented:
Merging this, especially since the Travis work is needed for any other PRs to be merged successfully and #594 is waiting on it.
